UnityProject/Assets/Scripts/UI/Cargo/CargoData.asset

```diff
@@ -95,7 +95,18 @@ MonoBehaviour:
       Items:
       - {fileID: 5313833756698493227, guid: 2d6f4fef721c0dc4787288d67494e06b, type: 3}
       - {fileID: 5313833756698493227, guid: 2d6f4fef721c0dc4787288d67494e06b, type: 3}
-  - CategoryName: Uniform Sets
+  - CategoryName: Science
+    Supplies:
+    - OrderName: Chemist kit Crate
+      CreditsCost: 800
+      Crate: {fileID: 7880601349656009979, guid: 907d16a099bbf496d9fc11b8c260ccf9,
+        type: 3}
+      Items:
+      - {fileID: 7785875528613525472, guid: f866b44bcdd7d734885f7c49999d85c9, type: 3}
+      - {fileID: 7785875528613525472, guid: f866b44bcdd7d734885f7c49999d85c9, type: 3}
+      - {fileID: 7374481989050852271, guid: 1fada2a01fe5e49208acb17e93095002, type: 3}
+      - {fileID: 7516689663430431150, guid: 8aa459a6447baed4a95b189f4950f59f, type: 3}
+      - CategoryName: Uniform Sets
```

Comment:
here was the problem. Fucked up the indentation, unity freaked and purged the entire category.

Reply:
did you actually modify asset file manually? they are only intended to be edited from editor, keep that in mind